test/main.js

```diff
@@ -1,7 +1,12 @@
 var globStream = require('../');
-var through2 = require('through2');
-var should = require('should');
+var miss = require('mississippi');
 var path = require('path');
+var should = require('should');
```

Comment:
Can you switch this to `expect` to align with the changes in master?  It will make things cleaner when I rebase.